data/aliases.js

```diff
@@ -872,79 +872,79 @@ let BattleAliases = {
 	"parasu": "Paras",
 	"parasekuto": "Parasect",
 	"konpan": "Venonat",
-	"morufon": "Venomoth",
-	"diguda": "Diglett",
+	"morufuon": "Venomoth",
+	"deiguda": "Diglett",
 	"dagutorio": "Dugtrio",
 	"nyaasu": "Meowth",
 	"perushian": "Persian",
 	"kodakku": "Psyduck",
 	"gorudakku": "Golduck",
-	"manki": "Mankey",
+	"mankii": "Mankey",
 	"okorizaru": "Primeape",
-	"gaadi": "Growlithe",
-	"uindi": "Arcanine",
+	"gaadei": "Growlithe",
+	"uindei": "Arcanine",
 	"nyoromo": "Poliwag",
 	"nyorozo": "Poliwhirl",
 	"nyorobon": "Poliwrath",
-	"keishi": "Abra",
+	"keeshyi": "Abra",
```

Comment:
This is awkward. Veekun uses `keesy` here. I might use `keeshii`? Bulbapedia uses `keeshy`.

Reply:
(official romaji is `casey`)

Reply:
I'm seeing a similar problem with Cleffa and Espeon – I suspect everywhere `yi` occurs.

Reply:
I'd probably go with Bulbapedia, just find/replace all the instances of `yi` with `y`.

Reply:
I went and replaced `yi` with `y`. Not sure its correct though? (Should serebyi really be sereby and not serebii?). Check daf3b0aa8.

Reply:
`serebii` is an alternative romanization I floated (I did say I'd personally use `keeshii` earlier), but it's destructive; i.e. it's ambiguous between ビー and ビィ.

Veekun uses `sereby` and Bulbapedia uses `serebyi`; I prefer Veekun because an English speaker would pronounce it closer to the correct pronunciation.

Reply:
Doing some more research: `serebii` is also misleading: it's not actually a long "i" sound; it's pronounced `serebi`.

Reply:
My only experience with Japanese is playing on some Japanese Pokemon cartridges as a kid who didn't want to wait for the North American releases and just ignoring all of the text, so I'm really glad you seem to understand what you're doing here!

Reply:
Heh, by coincidence, Japanese is my third best language and I spend a lot of time in Tokyo.